doc/deployment/deploy-to-aks-with-acr-using-jenkins.md

```diff
@@ -0,0 +1,152 @@
+# Deploy to Azure Kubernetes Service using Jenkins
+
+This document shows how to deploy this todo app java project to Kubernetes cluster using Jenkins.
```

Comment:
Add intro para: "You can queue an ACR quick build with GitHub URL. This document shows you how to use ACR build to build and create the Docker image in ACR."